EIPS/eip-867.md

```diff
@@ -0,0 +1,190 @@
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
+
+- People or organizations who are directly affected by the issue (e.g. entities who will be refunded or were associated with the original incident)
+- People or organizations that have a demonstrable history within the Ethereum community that are unaffected by the issue but may serve as relatively impartial adjudicators
+
+ERPs that have strong support from both groups of people will be much more likely to succeed.  The first group, although not impartial, is critical to determine that among the directly affected parties there is broad agreement about the desired outcome.  The second group is critical to help verify (a) the conditions defined in the verification script accurately represent the human readable version, (b) that the verification script generates the State Change Object submitted with the ERP, and (c) that the people/orgs representing the “affected parties” is representative.  
+
+Positive, explicit, and broad support from both groups will be critical to the approval process; a lack of negative feedback is not sufficient.  It is the sole responsibility of the ERP author and affected parties to find support. Feedback that is not substantive should be dismissed, and feedback for the ERP process as a whole should be directed to this meta-EIP.  Community members, in general, are encouraged to reference these guidelines to ensure the process remains focused, objective, and expedient.  
+
+The ERP author may indicate in the ERP comments when he/she believes there is sufficient support and request a review by an EIP editor (see EIP-1 for details).  The editor(s) will mark the ERP as “Accepted” or indicate why the proposal cannot be accepted.  An ERP that is not accepted may be re-evaluated upon substantial improvement.  
+
+Once an ERP meets the minimum requirements and client teams have indicated their intention to include the ERP as part of some future upgrade, the block at which the changes will be applied will be finalized and the ERP will be marked as "Accepted".  The target block will be determined based on coordination between the client teams and other pending releases.  Upon determination of the target block, a finalized version (updated to include the determined block) of the State Change Object will be published to the ERP.  Clients should bundle this final version (unmodified) with their respective applications.  
+
+Before the corresponding client updates are propagated on the mainnet, they should first be tested end-to-end on testnet networks, providing a final chance for the ERP stakeholders to monitor a test version of their recovery. Assuming the recovery action is included in an already planned network update, updated clients will already be tested on the testnet before a coordinated release on the mainnet. 
```

Comment:
Not only the ERP stakeholders, but also other network participants should be able to monitor the test version.  The ERP stakeholders are not on an isolated network.

Reply:
Good point.  I will review this section and I think including the state changes on the test net will not work, unless the situation is recreated on the test net and a test net version of the SCO object is created.